tapback/frontend.c

```diff
@@ -440,20 +440,15 @@ frontend_changed(vbd_t * const device, const XenbusState state)
 
     switch (state) {
         case XenbusStateInitialising:
-			if (device->hotplug_status_connected)
 				err = xenbus_switch_state(device, XenbusStateInitWait);
```

Comment:
Indentation.
Also, why does this need to be removed?
```
/local/domain/0/backend/vbd/<domid>/<vbdid>/hotplug-status = "connected"
```
Seems to be populated, is it happening too late?

Reply:
Yes, hotplug-status is populated after the frontend state is changed to connected (XenbusStateConnected)